language-server/src/main/java/org/sonarlint/languageserver/SonarLintLanguageServer.java

```diff
@@ -490,8 +488,25 @@ public WorkspaceService getWorkspaceService() {
 
   @Override
   public CompletableFuture<Object> executeCommand(ExecuteCommandParams params) {
-    // No command
-    return null;
+    switch (params.getCommand()) {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this "switch" statement by "if" statements to increase readability. ([squid:S1301](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1301))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&branch=feature%2Frule_description&issues=AWIAN8aaFcgpffuDkawq&open=AWIAN8aaFcgpffuDkawq)

[ISSUE_KEY]: AWIAN8aaFcgpffuDkawq